src/contactPage/contactPage.component.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  Theme,
+  StyleRules,
+  createStyles,
+  WithStyles,
+  withStyles,
+} from '@material-ui/core';
+import { getAppStrings, getString } from '../state/strings';
+import { connect } from 'react-redux';
+import { AppStrings } from '../state/scigateway.types';
+import { StateType } from '../state/state.types';
+
+const styles = (theme: Theme): StyleRules =>
+  createStyles({
+    root: {
+      padding: theme.spacing(2),
+      backgroundColor: theme.palette.background.default,
+      '& a': {
+        '&:link': {
+          color: '#1E5DF8',
```

Comment:
Could replace this with `(theme as UKRITheme).ukri.bright.blue` (and `purple`/`red` below).

Reply:
Wasn't aware that the theme existed but I like this idea.